roles/cdi/tasks/version.yml

```diff
@@ -0,0 +1,15 @@
+---
+# 'version_compare' changes to 'version' in ansible-2.5
+- name: "Setting cdi_release_tag to 0.5 if cdi_release_tag: {{ cdi_release_tag }} is >= 1.0 and kubevirt_version: {{ kubevirt_version }} < 0.7.0-alpha.4"
+  set_fact: cdi_release_tag="0.5"
+  when: "{{ '1.0' is version_compare(cdi_release_tag, '<=') }}
```

Comment:
Does this mean "when cdi_release_tag <= 1.0" or " when 1.0 <= cdi_release_tag"?  That is awful syntax.

Reply:
The syntax is really rough. This is when cdi >= 1.0 and kubevirt < 0.7.0-alpha.4 so cdi should be 0.5.
```
1.0 <= cdi_release_tag and 0.7.0-alpha.4 > kubevirt_version
```

Reply:
In-depth explanation is [here](https://github.com/kubevirt/kubevirt-ansible/pull/307#discussion_r199148911)